### ISSUE TITLE
Room cannot verify the data integrity.

### ISSUE THREAD

**Issue by RafaelBarbosatec** [giep-br/mobile-android#1]
I tried to update the AllIn SDK to this new version in the app I already have in production and when running the app it closes and presents this error:

```
E/AndroidRuntime: FATAL EXCEPTION: main
    Process: br.com.conectcar.debug, PID: 7467
    java.lang.RuntimeException: Unable to create application br.com.conectcar.evolution.DebugApplication: java.lang.IllegalStateException: Room cannot verify the data integrity. Looks like you've changed schema but forgot to update the version number. You can simply fix this by increasing the version number.
        at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6465)
        at android.app.ActivityThread.access$1300(ActivityThread.java:219)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1859)
        at android.os.Handler.dispatchMessage(Handler.java:107)
        at android.os.Looper.loop(Looper.java:214)
        at android.app.ActivityThread.main(ActivityThread.java:7356)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)
     Caused by: java.lang.IllegalStateException: Room cannot verify the data integrity. Looks like you've changed schema but forgot to update the version number. You can simply fix this by increasing the version number.
        at androidx.room.RoomOpenHelper.checkIdentity(RoomOpenHelper.java:154)
        at androidx.room.RoomOpenHelper.onOpen(RoomOpenHelper.java:135)
        at androidx.sqlite.db.framework.FrameworkSQLiteOpenHelper$OpenHelper.onOpen(FrameworkSQLiteOpenHelper.java:195)
        at android.database.sqlite.SQLiteOpenHelper.getDatabaseLocked(SQLiteOpenHelper.java:428)
        at android.database.sqlite.SQLiteOpenHelper.getWritableDatabase(SQLiteOpenHelper.java:317)
        at androidx.sqlite.db.framework.FrameworkSQLiteOpenHelper$OpenHelper.getWritableSupportDatabase(FrameworkSQLiteOpenHelper.java:145)
        at androidx.sqlite.db.framework.FrameworkSQLiteOpenHelper.getWritableDatabase(FrameworkSQLiteOpenHelper.java:106)
        at androidx.room.RoomDatabase.inTransaction(RoomDatabase.java:622)
        at androidx.room.RoomDatabase.assertNotSuspendingTransaction(RoomDatabase.java:399)
        at br.com.allin.mobile.pushnotification.dao.CacheDAO_Impl.get(CacheDAO_Impl.java:118)
        at br.com.allin.mobile.pushnotification.service.allin.CacheService.sync(CacheService.java:21)
        at br.com.allin.mobile.pushnotification.AlliNPush.registerForPushNotifications(AlliNPush.java:163)
        at br.com.allin.mobile.pushnotification.AlliNPush.registerForPushNotifications(AlliNPush.java:141)
        at br.com.conectcar.application.ConectCarApplication.onCreate(ConectCarApplication.java:60)
        at br.com.conectcar.evolution.DebugApplication.onCreate(DebugApplication.kt:21)
        at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1182)
        at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6460)
        at android.app.ActivityThread.access$1300(ActivityThread.java:219) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1859) 
        at android.os.Handler.dispatchMessage(Handler.java:107) 
        at android.os.Looper.loop(Looper.java:214) 
        at android.app.ActivityThread.main(ActivityThread.java:7356) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930) 
```

**Comment by RafaelBarbosatec** [comment]
Please increment database version in `AlliNDatabase` to 5

**Comment by lbrsilva-allin** [comment]
I released version 1.0.2, could you test it?

**Comment by RafaelBarbosatec** [comment]
Hi @lbrsilva-allin, problem resolved in 1.0.2 version.
Thanks!
